[en]: update network and storage pages (#4981)
DaoCloud/DaoCloud-docs: docs/en/docs/network/plans/netplan.md

# Network Planning

This page describes best practice in Overlay + Underlay CNI based on [Eth card planning](./ethplan.md), with the main scenarios covered is as follows:

1. Calico with Macvlan CNI
2. Calico with Macvlan and SR-IOV CNI

In the following scenarios, the best practice is to have 2 NICs, with different NICs carrying the following traffic:

1. **NIC 1 (eth0)**: Mainly responsible for Kubernetes internal management traffic, Calico traffic, and inter-node communication traffic
2. **NIC 2 (eth1)**: Mainly responsible for Underlay (Macvlan/SR-IOV) network traffic

## Calico with Macvlan CNI

![calico-macvlan](../images/calico-macvlan.jpg)

**Planning Instructions**:

- In this plan, the default CNI is Calico/Cilium, which needs to be installed together with components such as Multus-underlay and Spiderpool.
- It is recommended that all nodes have multiple physical network cards with the same name.

> - eth0 is the network card where the default route of the host is located, and the gateway points to the Gateway host, which forwards to the external network.

Reviewer: ```suggestion
- eth0 is a NIC where the default route of the host is located, and the gateway points to the Gateway host, which forwards to the external network.
```
Should be consistent with that on line 15 and 16 for `NIC`.

- The eth1.1, eth1.2 VLAN subinterfaces may not set IP addresses.

## Calico with Macvlan and SR-IOV CNI

![macvlan-sriov](..//images/macvlan-sriov.jpg)

**Planning Instructions**:

- In this plan, the default CNI is Calico/Cilium, which needs to be installed together with components such as Multus-underlay and Spiderpool.
- It is recommended that all nodes have multiple physical network cards with the same name.
- eth0 is the network card where the default route of the host is located, and the gateway points to the Gateway host, which forwards to the external network.
  The main uses are: communication between nodes, K8s management network card, Calico Pod communication.
- eth1 in `Worker01`、`Worker02` is the VLAN service network card and does not need to set an IP address.
  Create VLAN subinterfaces (eth1.1, eth1.2) based on eth1, corresponding to network segments such as 172.16.15.0/24 and 172.16.16.0/24.
  The created business application Pod uses the address of the corresponding network segment, which can meet the multi-VLAN and multi-subnet use cases.
- The eth1.1, eth1.2 VLAN subinterfaces may not have IP addresses.

> - eth1 in `Worker03` and `Worker04` is the SR-IOV service network card and does not need to set IP addresses. eth1 corresponds to 172.16.17.0/24,
> and eth1 can belong to the same network segment as eth1.1 and eth1.2, which can be decided according to the planning needs. 
> In this example, the network segments are staggered. The created business application Pod uses the corresponding network segment, 
> and based on the extended VF (Virtual Functions) can be directly invoked by the container to accelerate.

Reviewer: ```suggestion
- eth1 in `Worker03` and `Worker04` is a NIC for the SR-IOV service and does not need to be set IP addresses. eth1 corresponds to 172.16.17.0/24,
  and eth1 can belong to the same network segment as eth1.1 and eth1.2, which can be decided according to the planning needs. 
  In this example, the network segments are staggered. The created pod for business application uses the network segment, 
  and the extended VF (Virtual Functions) can be directly invoked by the container to get the acceleration effect.
```
长句都比较糟糕
